Sources/BraveVPN/BraveVPN.swift

```diff
@@ -49,7 +49,13 @@ public class BraveVPN {
     }
     
     if let customCredential = customCredential {
-      setCustomVPNCredential(customCredential.credential, environment: customCredential.environment)
+      if hasExpired == true {
+        clearConfiguration()
+        logAndStoreError("Skus credential expired, resetting configuration")
```

Comment:
If using cross platform credentials and the cred expired we clean all credentials and vpn configuration. The app will fallback to regular Apple's iAP

Reply:
Where is `hasExpired` defined?

Reply:
It just checkes the date we persist in `Preferences.VPN.expirationDate`

```
 public static var hasExpired: Bool? {
    guard let expirationDate = Preferences.VPN.expirationDate.value else { return nil }

    return expirationDate < Date()
  }
```